config/initializers/companies.yml

```diff
@@ -239,6 +249,7 @@
   address:    '20 Ayer Rajah Crescent, #08-07, Singapore 139964'
   hiring_url: 'https://jobs.lever.co/srcclr/'
   email:      'hello@srcclr.com'
+  contribution_count: 1
```

Comment:
@jsyeo: I would say talks at RDRC is already remunerated in one way or another, but let's ask @winston. 😀